testnets/build-requirements.md

```diff
@@ -21,27 +21,30 @@ Use docker image, or build locally: `https://github.com/CosmWasm/wasmd/#dockeriz
 
 ### Option 1 (preferred - build static binary using docker - Linux only)
 
-Constructing a `wasmd` is a tricky process and involves using alpine linux and recompiling the rust dependencies as static libs. Using Dockerfile is suggested.
+Constructing a `wasmd` is a tricky process and involves using Alpine Linux as a build environment and recompiling the Rust dependencies as static libs.
+Using Dockerfile is suggested.
 
-1. Clone the project `git clone git@github.com:CosmWasm/wasmd`
+1. Clone the project `git clone https://github.com/CosmWasm/wasmd.git && cd wasmd`
 2. Checkout to testnets version `git checkout vx.x.x`
 3. Build docker image `docker build . -t wasmd-docker`
 4. Extract the specific binary from docker:
 
-```sh
-id=$(docker create wasmd-docker)
-docker cp $id:/usr/bin/wasmd - > wasmd
-docker cp $id:/usr/bin/wasmcli - > wasmcli
-# docker cp $id:/usr/bin/corald - > corald
-# docker cp $id:/usr/bin/coralcli - > coralcli
-docker rm -v $id
-```
+   ```sh
+   id=$(docker create wasmd-docker)
+   docker cp $id:/usr/bin/wasmd - > wasmd
+   docker cp $id:/usr/bin/wasmcli - > wasmcli
+   chmod +x wasmd wasmcli
+   # docker cp $id:/usr/bin/corald - > corald
+   # docker cp $id:/usr/bin/coral - > coral
```

Comment:
Wait a second … docker cp seems to create a .tar file that needs to be extracted. Will update

Reply:
From `docker cp --help`:

> Use '-' as the destination to stream a tar archive of a container source to stdout.

Reply:
Done in https://github.com/CosmWasm/docs2/pull/54/commits/0fe82270dfa26c0a220078b7a3f6054fa892f0de